packages/react/src/components/Icon/index.tsx

```diff
@@ -14,10 +14,9 @@ export interface IconProps extends React.HTMLAttributes<HTMLDivElement> {
 const Icon = forwardRef<HTMLDivElement, IconProps>(
   ({ label, className, type, ...other }: IconProps, ref) => {
     const isMounted = useRef(true);
-    const [, name, direction] = type.match(/(.*)-(right|left|up|down)$/) || [
-      '',
-      type
-    ];
+    const [, name, direction] = (type !== 'table-sort-down' &&
+      type !== 'table-sort-up' &&
+      type.match(/(.*)-(right|left|up|down)$/)) || ['', type];
```

Comment:
Although this works, it would be nice to not have this override here. Instead of doing this, could we just change the name of the icon to not have "up" or "down"? For example, `table-sort-ascending` and `table-sort-descending`